pom.xml

```diff
@@ -464,5 +464,33 @@
 				</plugins>
 			</build>
 		</profile>
+		<profile>
+			<id>update_version_in_files</id>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>io.github.floverfelt</groupId>
+						<artifactId>find-and-replace-maven-plugin</artifactId>
+						<executions>
+							<execution>
+								<id>exec</id>
+								<phase>package</phase>
+								<goals>
+									<goal>find-and-replace</goal>
+								</goals>
+								<configuration>
+									<replacementType>file-contents</replacementType>
+									<baseDir>${project.basedir}</baseDir>
+									<findRegex>[0-9]+.[0-9]+.[0-9]+(-SNAPSHOT)?</findRegex>
+									<replaceValue>${project.version}</replaceValue>
+									<recursive>false</recursive>
+									<fileMask>README.md</fileMask>
```

Comment:
Avec ça, tu renommes juste le readme. Il va falloir une deuxième exécution pour renommer le fichier `.github\workflows\on_release_perform_maven_release.yml`

Reply:
On it